src/FlutureTMonetEither/index.js

```diff
@@ -366,7 +366,14 @@ FlutureTMonetEither.prototype.fork = function fork(leftFn, rightFn) {
  */
 FlutureTMonetEither.prototype.and = function and(futureEither) {
   return this.constructor.of(
-    this.run.and(futureEither.run)
+    Future.of(curry((e1, e2) => {
+      if (e1.isLeft()) {
```

Comment:
I would use monadic bind here and let Either deal with the binding implicitly. The if statements and this should be equivalent.
```javascript
return either1.chain(always(either2));
```